test/jdk/java/net/ipv6tests/Tests.java

```diff
@@ -123,7 +123,8 @@ public static void datagramEcho (DatagramSocket s1, DatagramSocket s2,
         }
 
         dprintln ("dest2 = " + dest2);
-
+        dprintln ("sender endpoint == " + s1.getLocalSocketAddress());
+        dprintln ("echo endpoint == " + s2.getLocalSocketAddress());
```

Comment:
Nit: Any reason to have a single " = " line 125 and two " == " on these two lines?

Reply:
not really, changes to single = as in the rest of outputs